[refactor]반려견 페이지 수정

JYeyun opened the pull request:
- mypage 반려견 추가 삭제
- mypage 나의 반려견 정보 페이지로 통합
- 반려견 리스트, 반려견 detail 페이지 fragment로 변경
- viewModel 리펙토링
- add, edit 시 카메라 버튼 추가

resolve : #238

JYeyun committed:
[refactor]반려견 페이지 수정

- mypage 반려견 추가 삭제
- mypage 나의 반려견 정보 페이지로 통합
- 반려견 리스트, 반려견 detail 페이지 fragment로 변경
- viewModel 리펙토링
- add, edit 시 카메라 버튼 추가

#238

neoneoneo123 reviewed (changes requested):
확인했습니다.

세 가지 코멘트가 있습니다!

1. 반려견 추가 시 "남자""여자" 텍스트가 사용되고 있습니다.
<img width="245" alt="image" src="https://github.com/nbc-final-team2/ddaraogae/assets/128209823/ba42f3c9-7fec-4a3b-9129-ec1a60a0b519">

2. detailPetFragment에서 화면 회전 시 PetListFragment로 이동하고 있습니다.
![Jul-01-2024 10-21-14](https://github.com/nbc-final-team2/ddaraogae/assets/128209823/2d6021a0-77b8-49e3-bfcc-7e20dcbdd32c)


4. 반려견 삭제 후 뒤로가기 버튼 클릭 시 삭제한 반려견의 정보 화면으로 이동합니다.
![Jul-01-2024 10-20-19](https://github.com/nbc-final-team2/ddaraogae/assets/128209823/27215abb-454e-4641-83bb-e660a3c2557b)

IamJuco reviewed (changes requested):
코멘트 남깁니다!

![Screenshot_20240701_114703](https://github.com/nbc-final-team2/ddaraogae/assets/43128821/e7c389da-a058-4bcf-81d9-deba1e3be764)

1. 위 사진처럼 앱상태바와 네비게이션바에 기본 color값이 나오고있습니다.
activity_my_pet.xml에 background color = white값을 주면 해결될듯 합니다!

2. 읽기전용 페이지 이지만 남아, 여자가 선택된 곳이 너무 밋밋한 느낌이 드는것 같습니다.
추가 할때 처럼 shape 속성 추가 하면 좋지 않을까 싶습니다!

JYeyun committed:
[fix] comment 수정

#238

- 반려견 추가 페이지 남자, 여자 텍스트 수정
- detailPetFragment에서 화면 회전 시 PetListFragment로 이동하지 않도록 조치
- 수정하기 페이지 종료 시 강아지 리스트 화면으로 이동하도록 수정
- my_pet.xml background 수정
- 강아지 정보 창 남아 여아 테두리 추가

JYeyun commented:
- 반려견 추가 페이지 남자, 여자 텍스트 수정
- detailPetFragment에서 화면 회전 시 PetListFragment로 이동하지 않도록 조치
- 수정하기 페이지 종료 시 강아지 리스트 화면으로 이동하도록 수정
- my_pet.xml background 수정
- 강아지 정보 창 남아 여아 테두리 추가
e8c312dbe4f19c4b8b275ad17fda7e1fb4d4dce5

neoneoneo123 reviewed (approved):
확인했습니다!
고생하셨습니다~

IamJuco reviewed (approved):
확인 했습니다
수고하셨습니다!

JYeyun committed:
Merge branch 'develop' into refactor/mypage-dog-detail
